media/inf/unknown.inf

```diff
@@ -49,3 +49,7 @@ UnknownClassName = "Başka Aygıtlar"
 [Strings.0422]
 ReactOS = "Команда ReactOS"
 UnknownClassName = "Інші пристрої"
+
+[Strings.0804]
+ReactOS = "ReactOS Team"
```

Comment:
You might wanna make this word Chinese?

Reply:
From the PR comment:
> In this commit, the string "ReactOS 团队" was restored to "ReactOS Team" because it is not in line with Simplified Chinese IT language habits.